mlptrain/sampling/reaction_coord.py

```diff
@@ -122,3 +128,82 @@ def _grad(self, atoms: ase.atoms.Atoms):
     def n_pairs(self) -> int:
         """Number of pairs of atoms in this set"""
         return len(self.atom_pair_list)
+
+
+class DifferenceDistance(ReactionCoordinate):
+    """Difference in distance between two pairs of specified atoms"""
+
+    def __init__(self, *args):
```

Comment:
Given the constructor is basically identical for this and `AverageDistance` I would maybe have a `_TwoDIstances` base class